pulpcore/pulpcore/app/apps.py

```diff
@@ -111,6 +116,14 @@ def import_viewsets(self):
                     # obj isn't a class, issubclass exploded but obj can be safely filtered out
                     continue
 
+    def import_urls(self):
+        """
+        If a plugin defines a urls.py, include it.
+        """
+        if module_has_submodule(self.module, URLS_MODULE_NAME) and self.name != "pulpcore.app":
+            urls_module_name = '%s.%s' % (self.name, URLS_MODULE_NAME)
```

Comment:
Total nitpick, but I generally prefer ```.format()```.  This way is fine though

Reply:
I agree with you, but since this code is almost identical to the others, I'd like to leave it to highlight the similarity.